### DISCUSSION TITLE
tracking goals

### DISCUSSION THREAD

**Discussion by damienservais** [queen-raae/gatsby-plugin-fathom#11] [Q&A] [answered]
Hi, thanks for the plugin. How do you track goals?

I need to call this function, do you know how to get fathom.trackGoal() ?
I'm looking for something similar to the other plugin like (import { useGoal } from 'gatsby-plugin-fathom'))

Thanks you

**Comment by raae** [comment]
Thanks for your interest, 

I did not get to that far, but I do plan to do some more work.

However gatsby-plugin-fathom was reactivated (after I did this work) so you could probably just use that one. 

Is gatsby-plugin-fathom not working for you?

**Reply by raae** [reply]
Even looks like I got as far as a PR (#3), how does that solution work for you?

**Comment by damienservais** [comment]
Thanks, yes it looks exactly like what I’m looking for. I have some issues to install the other plugin.

**Reply by raae** [reply]
I'll get on it 🚀

**Comment by damienservais** [comment]
great, thanks you very much!

**Comment by raae** [comment, marked as answer]
Only took about a year 🤪
Released in v0.1.0
